.travis.yml

```diff
@@ -0,0 +1,82 @@
+---
+version: ~> 1.0
+conditions: v1
+dist: bionic
+# Config file for automatic testing at travis-ci.org
+
+services:
```

Comment:
we don't need docker here

Reply:
well, we need it for tox -e distros, added comment explaining why.